packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -157,6 +157,13 @@ const patternSource = computed(() => {
       $vertical-padding: var(--onyx-input-padding-vertical)
     );
 
+    &__header {
```

Comment:
This class does not exist, right? Same for the textarea?

Reply:
The class is now part of the `OnyxFormElement`, but these changes have therefore become redundant here.